testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -12472,6 +12472,21 @@ void testTimestampDiff(boolean coercionEnabled) {
     f.checkScalar("FORMAT_DATE('The date is: %x', DATE '2008-12-25')",
         "The date is: 12/25/08",
         "VARCHAR(2000) NOT NULL");
+    f.checkScalar("FORMAT_DATE('%G-%V', DATE '2023-01-01')",
```

Comment:
same comment about dates before the Gregorian calendar

Reply:
Done. I also had to fix one test case for PostgreSQL ISOWEEK - also manually validated using dbfiddle.